packages-rhcos.yaml

```diff
@@ -15,7 +15,11 @@ packages:
   - nss-altfiles
   - ostree
   - rpm-ostree
-  - toolbox
   # for kdump:
   # https://github.com/coreos/fedora-coreos-tracker/issues/622
   - kexec-tools
+
+repo-packages:
```

Comment:
I don't think that it's a key supported by the apply-manifest script yet.

Reply:
@travier I built it locally and it seemed to work.  I'll have to move it up into `manifest-ocp-rhel-9.6.yaml` because our CI is testing 9.4 (9.6 in the works https://github.com/openshift/os/pull/1703) and adding `repo-packages` in the `package-rhcos.yaml` is making the 9.4 CI look in the `rhel-9.6-server-ose-4.19` repo which isn't defined for 9.4 CI.